model/src/main/kotlin/config/ScopeExcludeReason.kt

```diff
@@ -24,19 +24,48 @@ package com.here.ort.model.config
  */
 enum class ScopeExcludeReason {
     /**
+     * @deprecated 
```

Comment:
To mark something as deprecated in Kotlin, use `@Deprecated` (captitalized) as part of the code, not the docs, and name the new symbol to use. Like:

    @Deprecated(
        message = "Use SPDX-2.2-style enum value instead.",
        replaceWith = ReplaceWith(
            expression = "ScopeExcludeReason.BUILD_DEPENDENCY_OF",
            imports = ["com.here.ort.model.config.ScopeExcludeReason"]
        )
    )
    BUILD_TOOL_OF,

Reply:
Fixed